deploy/kube.yaml

```diff
@@ -23,6 +23,7 @@ metadata:
   annotations:
     ingress.kubernetes.io/auth-url: http://authn-stub.atlas-stubs.svc.cluster.local/auth
     ingress.kubernetes.io/auth-response-headers: Grpc-Metadata-Authorization
+    nginx.ingress.kubernetes.io/rewrite-target: /
```

Comment:
with this rewrite target, all requests coming in from the api gateway will resolve to the mux without the `atlas-contacts-app` prefix. This is nice because it decouples the ingress registration and the server's mux